app/views/exportImport.html

```diff
@@ -0,0 +1,130 @@
+<div class="page-content">
+  <div class="row">
+    <!-- include the sidebar -->
+    <div ng-include="'views/sidebar.html'"></div>
+    <div class="col-md-10">
+      <div class="content-box-large">
+        <div class="panel-heading">
+          <div class="panel-title">
+            <h2><i class="glyphicon glyphicon-transfer"></i>&nbsp;&nbsp;Export/Import</h2>
+          </div>
+        </div>
+
+        <!-- Top Alerts -->
+        <alert ng-repeat="alert in alerts.top" type="alert.type">{{alert.msg}}</alert>
+        <!-- Top Alerts -->
+        
+        <div class="exportSectionDiv">
+
+          <h3><strong>Export your configurations</strong></h3>
+          <div style=" float: left; clear: both; width: 100%; margin-bottom: 10px;">
+            <label tooltip="For a 'Insert Script' all '_id' properties will be removed and re-created on import">Will this export file be used as an Insert or Update?</label><br />
+
+            <div class="btn-group" style="margin-bottom: 10px;">
+              <label class="btn btn-primary" ng-model="exportSettings.removeIds" btn-radio="true"><strong>Insert</strong> Script</label>
+              <label class="btn btn-primary" ng-model="exportSettings.removeIds" btn-radio="false"><strong>Update</strong> Script</label>
+            </div>
+          </div>
+          
+          
+          <div class="exportCollectionsDiv" ng-repeat="(model, collectionData) in exportCollections">
+          
+            <label class="exportModelTitle">All {{model}} <input type="checkbox" ng-checked="true" ng-model="checkBox" value="{{collectionData}}" ng-change="toggleCollectionExportSelection(model, collectionData)" /></label>
```

Comment:
I think moving the checkboxes to the front of the text will look better. For both the All {{model}} and the individual records.